taskfiles/docker.yml

```diff
@@ -7,6 +7,19 @@ tasks:
     status:
     - test -f bin/task
 
+  stop-containers:
```

Comment:
this fixes ducky?

Reply:
it remains to be seen... I have to test the specific scenario of running a job on reused agents (job running after failed/cancelled builds)

Reply:
that's on https://github.com/vectorizedio/vtools/pull/210